bin/test_validator

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash -e
+
+ROOT_DIR=$(dirname $0)/..
+cd $ROOT_DIR
+
+if [[ $# != 1 ]]; then
+    echo Usage: $0 PROJECT_ID
+    false
+fi
+project_id=$1
+shift
+
+site_path=udmi_site_model
+device_id=AHU-1
+
+serial_no=validator-$RANDOM
+echo Using pubber with serial $serial_no
+
+PUBBER_OUT=pubber.out
+
+pids=`ps ax | fgrep pubber | fgrep java | awk '{print $1}'`
+if [[ -n $pids ]]; then
+    echo Killing pubber pids $pids
+    kill $pids
+fi
+
+rm -rf out/devices
+
+# Prepare auth key used by reflector
+cp udmi_site_model/devices/AHU-1/rsa_private.pkcs8 \
+   udmi_site_model/gcp_reflect_key.pkcs8
+
+bin/validator $site_path $project_id &
```

Comment:
Should we trap on exit of validator so we know if it exits while running pubber?

Reply:
Maybe-- the code as it stands checks the output of what validator does, so it doesn't matter if it exits prematurely or not... Either the generated output is correct or not.